src/GitVersionCore.Tests/VariableProviderTests.cs

```diff
@@ -154,6 +154,8 @@ public void ProvidesVariablesInContinuousDeploymentModeForStableWhenCurrentCommi
     }
 
     [Test]
+    [Category("NoMono")]
+    [Description("Won't run on Mono due to source information not being available for ShouldMatchApproved.")]
```

Comment:
[Seems like the build ran on Mono just fine](https://travis-ci.org/GitTools/GitVersion/builds/137612887).